flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/TaskExecutor.java

```diff
@@ -641,7 +641,7 @@ private void stopTaskExecutorServices() throws Exception {
 	}
 
 	@Override
-	public void releasePartitions(Collection<ResultPartitionID> partitionIds) {
+	public void releasePartitions(JobID jobId, Collection<ResultPartitionID> partitionIds) {
```

Comment:
a bit weird that we will leave `jobId` here unused, is the future change that needs it going to be so big?

Reply:
it's not gonna be big, but this part is simply already done.